src/components/skills-quiz-v2/styles/_Body.scss

```diff
@@ -1,37 +1,42 @@
-$gray-500: #707070;
-$max-sm-modal-width: 28rem !important;
-$min-sm-modal-height: 16rem !important;
-
-.page-body {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-}
-.form {
-  width: 65%;
-}
-.pgn__form-group {
-  width: 101%;
-}
-.text {
-  width: 65%;
-}
-.pgn__form-label {
-  font-weight: bold;
-}
+.skills-quiz-v2 {
+  $max-sm-modal-width: 28rem !important;
+  $min-sm-modal-height: 16rem !important;
 
-.btn {
-  margin: 7px -5px;
-}
+  .page-body {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
 
-.form-auto-suggest {
-  .form-control {
-    color: $gray-500;
+  .form {
+    width: 65%;
+  }
+
+  .pgn__form-group {
+    width: 101%;
   }
-}
 
-.modal-small {
-  max-width: $max-sm-modal-width;
-  min-height: $min-sm-modal-height;
+  .text {
+    width: 65%;
+  }
+
+  .pgn__form-label {
+    font-weight: bold;
+  }
+
+  .advanced-options-btn {
+    text-decoration: none;
```

Comment:
nit: No need to define custom CSS class for removing text-decoration from an element. There is a `.text-decoration-none` CSS utility class name that can be used instead. See https://paragon-openedx.netlify.app/foundations/css-utilities/#css-utilities.